### PR TITLE
Some suggestions and amendments

### PR DESCRIPTION
**Pull request by henkvancann**
I stopped after a while, checked the result locally.  
Please let me know whether the teams appreciates this kind of contribution.

### CODE REVIEW

File: spec/terminology.md [hunk 1]
```diff
@@ -10,10 +10,11 @@
 ~ As defined by [DID Core](https://www.w3.org/TR/did-core/#dfn-did-documents). See section [DID Documents](#did-documents).
 
 [[def: key event log, KEL]]
-~ As defined by KERI.
+~ As defined by [KERI](https://github.com/WebOfTrust/WOT-terms/wiki/key-event-log).
 
 [[def: transaction event log, TEL]]
-~ As defined by KERI.
+~ As defined by [KERI](https://github.com/WebOfTrust/WOT-terms/wiki/transaction-event-log).
```
Comment (swcurran) [inline spec/terminology.md:16]: I think this should be a [[ref:..]], with a corresponding [[def: ]] that has a brief description and link. The benefit of that is that in the rendered spec, the link is a mouseover and a minimum definition can be obtained without leaving the spec — unless the reader wants to.

**Comment by swcurran** [comment]
Updates like this are indeed welcome.  Please make sure to sign (lower case -c) your commits with DCO — DCO - Developer Certificate of Origin - https://github.com/apps/dco. They must be signed to be accepted/merged.

File: spec/terminology.md (hunk 1 above)
Comment (henkvancann) [reply spec/terminology.md:16]: > I think this should be a [[ref:..]], with a corresponding [[def: ]] that has a brief description and link. The benefit of that is that in the rendered spec, the link is a mouseover and a minimum definition can be obtained without leaving the spec — unless the reader wants to.

Ouch, I normally do, have to look into what went wrong.

Comment (henkvancann) [reply spec/terminology.md:16]: I default commit as 'verified'. 
Screenshot 2023-10-06 at 08 49 36

I see that DCO is something else / extra. I'll go and try to add this data to my commits

Comment (henkvancann) [reply spec/terminology.md:16]: Done

Comment (henkvancann) [reply spec/terminology.md:16]: > I think this should be a [[ref:..]], with a corresponding [[def: ]] that has a brief description and link. The benefit of that is that in the rendered spec, the link is a mouseover and a minimum definition can be obtained without leaving the spec — unless the reader wants to.

Done, great point: stay in the spec.

File: spec/abstract.md [hunk 2]
```diff
@@ -16,15 +16,15 @@ publication mechanism. This offers an interoperability bridge from (or between)
 blockchain ecosystems. Also, without directly supporting environments where the
 web is not practical (e.g., IOT, Lo-Ra, BlueTooth, NFC), the method builds on a
 foundation that fully supports those environments, making future interop of
-identifiers between web and non-web an easy step.
-
+identifiers between web and non-web a manageable step.
+<!-- Nothing is 'easy' in our line of work -->
```
Comment (2byrds) [inline spec/abstract.md:20]: I assume we won't include this line, it's just a comment for our review?

**Review by 2byrds — CHANGES_REQUESTED** [review]

**Comment by 2byrds** [comment]
Great progress @henkvancann ! Looking forward to strengthening it together!

File: spec/core.md [hunk 3]
```diff
@@ -62,15 +63,16 @@ when the domain name portion of one of its DIDs is resolved through the Domain
...
+DID document, except for minor differences in the `id`, `controller`, and `alsoKnownAs` top-level properties that pertain to the identifiers themselves.
 
 However, this method introduces an important nuance about the target system. In
 many DID methods, the target system equals a [[ref: verifiable data registry]] —
 besides publishing data, its security attributes make that data trustworthy. In
 this DID method, the target system's role is more limited. It is expected to
 serve data about the DID, and to follow acknowledged cybersecurity best
-practices to preserve good hygiene. However, the authenticity of data is
+practices to preserve good hygiene.
+<!-- What are these acknowledges rules and what is good hygiene? -->
```
[the remark this answers is not in the document]
Comment (henkvancann) [reply spec/core.md:74]: > I assume this won't be included, 
Right, I suggest we clear all my html comments out: 

> and that we can discuss them some time for better clarification. Consider using the github comments for discussion.
Right, if I knew how in combination with PRs... Could you give an example?
> In terms of good hygiene.... thats a long topic that we probably need to handwave to keep the spec tight.
Yes, there are more parts in the current specs, that I would shorten drastically and then link to a spot where there is an elaboration. Any ideas how to do this? My 2 cents: We could create a small more_on_|topic|.md per topic.

File: spec/abstract.md (hunk 2 above)
Comment (henkvancann) [reply spec/abstract.md:20]: Yes

File: spec/abstract.md [hunk 4]
```diff
@@ -16,15 +16,15 @@ publication mechanism. This offers an interoperability bridge from (or between)
 blockchain ecosystems. Also, without directly supporting environments where the
 web is not practical (e.g., IOT, Lo-Ra, BlueTooth, NFC), the method builds on a
 foundation that fully supports those environments, making future interop of
-identifiers between web and non-web an easy step.
-
+identifiers between web and non-web a manageable step.
```
[the remark this answers is not in the document]
Comment (henkvancann) [reply spec/abstract.md:19]: Not sure if I get what you exactly mean, I suggest a brief Zoom call to instruct me (see Discord PM).

[the remark this answers is not in the document]
Comment (2byrds) [reply spec/abstract.md:19]: Thank you for the zoom meeting to discuss how to move forward. Looking forward to these PRs.

File: spec/introduction.md [hunk 5]
```diff
@@ -11,4 +11,6 @@ Unfortunately, that answer works better for the first question than the second.
...
 
-The `did:webs` method described in this spec separates these two questions, and gives them different answers. _Information about DIDs_ is still published on the web, but _its trustworthiness_ derives from mechanisms entirely governed by individual DID controllers. This preserves most of the delightful convenience of `did:web`, while drastically upgrading security and decentralized trust.
\ No newline at end of file
+The `did:webs` method described in this spec separates these two questions and answers them distinctively. _Information about DIDs_ is still published on the web, but _its trustworthiness_ derives from mechanisms entirely governed by individual DID controllers. This preserves most of the delightful convenience of `did:web`, while drastically upgrading security and decentralized trust.
+<!-- What is decentralized trust? We might need to describe edge cases that are either in or out this container term -->
```
[the remark this answers is not in the document]
Comment (henkvancann) [reply spec/introduction.md:15]: done

File: spec/core.md [hunk 6]
```diff
@@ -27,6 +27,7 @@ The ABNF definition of a `did:webs` DID is as follows:
 webs-did = "did:webs:" domain-name ":" aid
 webs-did = "did:webs:" domain-name * (":" path) ":" aid
 ```
+`ABNF` stands for Augmented Backus-Naur Form, which is a formal notation used for describing syntax rules in various kinds of languages and protocols. We need to ensure that our implementation aligns with W3C standards and the ABNF rules specified in the DID documentation. These rules serve as a kind of "grammar" for DIDs, ensuring that they are formatted and used consistently.
```
Comment (2byrds) [inline spec/core.md:30]: If DID core refers to ABNF, then we should link there instead of adding this paragraph. Can you double check?

Comment (henkvancann) [reply spec/core.md:30]: Yes, that's what I intend to do: one sentence + link. Some terms are just too complicated to define in one sentence. But it's what I strive for. See my Termtext PR72.
Btw: Feel free to add / change stuff you don't like afterwards. It's getting messy now with too may PRs, please accept a few first / then modify according to your wishes and then I'll be back on the front ;)

**Review by swcurran — APPROVED** [review]
Looks good to me. In reading the text I still see more things I would do, but lets get this merged and do more passes :-)

Thanks!

File: spec/core.md [hunk 7]
```diff
@@ -230,13 +232,13 @@ Creating a `did:webs` DID involves the following steps:
...
-5. Transform the derived `did:webs` DID document to the corresponding `did:web` DID document according
-   to section [Transformation to did:web DID Document](#transformation-to-didweb-did-document).
+5. For compatibility reasons, transform the derived `did:webs` DID document to the corresponding `did:web` DID document according to section [Transformation to did:web DID Document](#transformation-to-didweb-did-document).
 6. Create the AID folder on the web server at the selected location, and place
    the `did:web` DID document resource (`did.json`) and the [[ref: KERI event stream]] resource (`keri.cesr`)
    into that folder. See section [Target System(s)](#target-systems) for further details about
    the locations of these resources.
-
+<!-- Why can't 2. be " Create a KERI AID and add it as the last element of the web URL for the DID" and 3. "Add the appropriate KERI events to the AID's KERI logs that will correspond to properties of the DID document, such as verification methods and service endpoints" ? The AID won't change if we add events to KEL/TEL. I am not 100% sure about this though. -->
```
[the remark this answers is not in the document]
Comment (henkvancann) [reply spec/core.md]: I added it in commit https://github.com/trustoverip/tswg-did-method-webs-specification/pull/65/commits/68ab0a6e932d3676cfb2ac05923d04eba6642618

**Comment by dhh1128** [comment]
I also vote to merge now.

**Review by 2byrds — APPROVED** [review]
Thank you for the updates!

**Pull request merged**